.github/workflows/go-cross.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [ 1.19, 1.x ]
+        go-version: [ 1.23, 1.x ]
```

Comment:
```suggestion
        go-version: [ oldstable, stable ]
```

Reply:
I noticed it does not work with `actions/setup-go@v2`.
updated with https://github.com/traefik/plugindemo/pull/24/commits/db60eba23cf607c4fef040b5e6f0210f48620be7 :)